tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -307,15 +308,24 @@ func newPromProxy(namespace string) *promProxy {
 func dumpK8Env() {
 	_, _ = util.Shell("kubectl --namespace %s get pods -o wide", tc.Kube.Namespace)
 
-	podLogs("istio="+ingressName, ingressName)
-	podLogs("istio=mixer", "mixer")
-	podLogs("istio=pilot", "discovery")
-	podLogs("app=productpage", "istio-proxy")
+	podLogs("istio="+ingressName, ingressName, tc.Kube.KubeConfig)
+	podLogs("istio=mixer", "mixer", tc.Kube.KubeConfig)
+	podLogs("istio=pilot", "discovery", tc.Kube.KubeConfig)
+	podLogs("app=productpage", "istio-proxy", tc.Kube.KubeConfig)
+	if tc.Kube.RemoteKubeConfig != "" {
+		_, _ = util.Shell("kubectl --namespace %s get pods -o wide --kubeconfig=%s", tc.Kube.Namespace, tc.Kube.RemoteKubeConfig)
+	}
 
 }
 
-func podID(labelSelector string) (pod string, err error) {
-	pod, err = util.Shell("kubectl -n %s get pod -l %s -o jsonpath='{.items[0].metadata.name}'", tc.Kube.Namespace, labelSelector)
+func podID(labelSelector string, remote bool) (pod string, err error) {
```

Comment:
consider changing variable name 'remote' to 'pod_location' or something other than 'remote' so that below you can use
if pod_location == remote {

Reply:
If you do change passed in parameter name to something other than 'remote' (seems a bit odd when you think of the calling function using 'local') then the last function in this change set also should be updated.